contracts/l2/gateway/L2GraphTokenGateway.sol

```diff
@@ -238,8 +238,7 @@ contract L2GraphTokenGateway is GraphTokenGateway, L2ArbitrumMessenger, Reentran
         if (_data.length > 0) {
             bytes memory callhookData;
             {
-                bytes memory gatewayData;
-                (gatewayData, callhookData) = abi.decode(_data, (bytes, bytes));
+                (, callhookData) = abi.decode(_data, (bytes, bytes));
```

Comment:
@pcarranzav do you remember the content of gatewayData? where was this coming from?

Reply:
Yeah that data is just sent by the Arbitrum router when creating the default bridge + L2 tokens, so we don't need it but we keep the format for compatibility

Reply:
So in more detail:

The data is decoded here: https://github.com/OffchainLabs/arbitrum/blob/master/packages/arb-bridge-peripherals/contracts/tokenbridge/arbitrum/gateway/L2ArbitrumGateway.sol#L235-L255

And then used here: https://github.com/OffchainLabs/arbitrum/blob/master/packages/arb-bridge-peripherals/contracts/tokenbridge/arbitrum/gateway/L2ERC20Gateway.sol#L79-L102

Which then initializes the L2 token here: https://github.com/OffchainLabs/arbitrum/blob/288488a64d0510d7eb7f9f83b346fa00feb1bb02/packages/arb-bridge-peripherals/contracts/tokenbridge/arbitrum/StandardArbERC20.sol#L44-L80